.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,10 +3,11 @@
 <!-- Please read CONTRIBUTING.md before submitting your pull request -->
```

Comment:
Can we also change this comment to refer to `https://matrix-org.github.io/synapse/latest/development/contributing_guide.html`)?